sdk/servicebus/service-bus/test/stress/Dockerfile

```diff
@@ -1,8 +1,9 @@
 FROM node:8-alpine
 RUN mkdir -p /src
-ADD ./*.ts /src
-ADD  ./temp.package.json /src/package.json
-ADD  ./tsconfig.json /src/tsconfig.json
-RUN cd /src && npm install && npm run build
-WORKDIR /src
+ADD ./src/*.ts /src
```

Comment:
We probably don't want to add it to the root, right?

We could rename the top folder to /app if that's what you were trying to fix. Then there would just be:

```
/app
/app/src/*.ts
/app/package.json
/app/tsconfig.json
```